server/sonar-web/src/main/js/apps/issues/components/BulkChangeModal.js

```diff
@@ -142,12 +145,29 @@ export default class BulkChangeModal extends React.PureComponent {
     this.props.onClose();
   };
 
-  handleAssigneeSearch = (query /*: string */) => searchAssignees(query, this.state.organization);
+  handleAssigneeSearch = (query /*: string */) => {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove curly braces and "return" from this arrow function body. ([javascript:S3524](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fgreg%2Fhardening-26-02&issues=AWHS551cn7Gi3ZonCdQm&open=AWHS551cn7Gi3ZonCdQm)

[ISSUE_KEY]: AWHS551cn7Gi3ZonCdQm